src/go/rpk/pkg/cli/cmd/wasm/generate.go

```diff
@@ -42,23 +42,29 @@ func NewGenerateCommand(fs afero.Fs) *cobra.Command {
 	var output string
 	const localDir = "./"
 	command := &cobra.Command{
-		Use:   "generate",
-		Short: "Create a npm template project for inline WASM engine",
+		Use:	"generate <name>",
```

Comment:
So what's `--out` for?

Reply:
`--out-dir` is the dir where we put the npm project, that folder is called with `<name>`